metricflow/test/model/validations/test_data_warehouse_tasks.py

```diff
@@ -152,26 +126,20 @@ def test_validate_dimensions(  # noqa: D
     issues = dw_validator.validate_dimensions(model)
     # One isssue is created for the short circuit query failure, and another is
     # created for the sub task checking the specific dimension
-    print("\nModelValidationResults", issues)
     assert len(issues.all_issues) == 2
-    assert (
-        f"Failed to query dimensions in data warehouse for data source `{model.data_sources[0].name}`"
-        in issues.all_issues[0].message
-    )
-    assert (
-        f"Unable to query `doesnt_exist` in data warehouse for dimension `doesnt_exist` on data source `{model.data_sources[0].name}`"
-        in issues.all_issues[1].message
-    )
```

Comment:
The issues all generate the same basic message anyway regardless of the failure cause, right? If not, we could convert these to more generic error message assertions, or scan the full set for each failing data source, but if so removing this makes more sense.